### PR TITLE
Added words_to_watch to feature_lists/ptwiki.py

### PR DESCRIPTION
**Pull request by chtnnh**

### CODE REVIEW

File: articlequality/feature_lists/ptwiki.py [hunk 1]
```diff
@@ -177,7 +180,9 @@ def filter_paragraphs_without_ref_tags(segment):
     main_article_templates / max(wikitext.revision.content_chars, 1),
     (portuguese.stemmed.revision.stem_chars /
      max(wikitext.revision.content_chars, 1)),
-    log(paragraphs_without_refs_total_length + 1)
+    log(paragraphs_without_refs_total_length + 1),
+    words_to_watch_count,
+    words_to_watch_count / max(wikitext.revision.words, 1)
```
Comment (he7d3r) [inline articlequality/feature_lists/ptwiki.py:185]: What do you think about adding the ratio `words_to_watch_count / max(all_cite_templates, 1)`? Maybe it captures the idea that the presence of words to watch in the text is more problematic when there are few citations than when there are many.

Comment (halfak) [reply articlequality/feature_lists/ptwiki.py:185]: I think that is a great idea!

File: model_info/ptwiki.wp10.md [hunk 2]
```diff
@@ -15,71 +15,71 @@ Model Information:
 	 - python_implementation: 'CPython'
 	 - python_revision: ''
 	 - python_version: '3.5.3'
-	 - release: '4.9.0-11-amd64'
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=8921):
+	counts (n=8574):
```
Comment (halfak) [inline model_info/ptwiki.wp10.md:21]: This has dropped a lot.  I wonder if it is related to some pages getting deleted?

Comment (halfak) [reply model_info/ptwiki.wp10.md:21]: I confirmed that https://github.com/wikimedia/articlequality/blob/master/articlequality/utilities/fetch_text.py#L110 does follow redirects so pagemoves shouldn't case a big loss in observations.

File: model_info/ptwiki.wp10.md [hunk 3]
```diff
@@ -15,71 +15,71 @@ Model Information:
...
-		'2'      1494  -->    90  1057   300    21    21     5
-		'3'      1497  -->    31    94  1066   166    97    43
-		'4'      1489  -->    15    31   296   332   451   364
-		'5'      1483  -->    10    12   116   227   750   368
-		'6'      1463  -->    14    17    50   150   293   939
+		'1'      1489  -->  1261   129    87     3     7     2
+		'2'      1497  -->   116  1040   301    15    19     6
+		'3'      1494  -->    48    87  1048   190    70    51
+		'4'      1489  -->    19    28   235   409   393   405
+		'5'      1210  -->    12    17    66   259   619   237
```
Comment (halfak) [inline model_info/ptwiki.wp10.md:28]: Lots of level 5 pages missing.  This is weird.

**Comment by halfak** [comment]
Looks like we can demonstrate some gains.  See https://www.mediawiki.org/w/index.php?title=ORES/Issues/Article_quality&oldid=3860116#Summary for analysis.  Merging.

**Pull request merged**